packages/flutter/lib/src/material/dialog.dart

```diff
@@ -52,6 +53,11 @@ class Dialog extends StatelessWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// The z-coordinate of this [Dialog].
+  ///
```

Comment:
Explain how the default elevation is computed here.

Reply:
The [dartdoc doc for macros](https://github.com/dart-lang/dartdoc) doesn't mention nested macros, so it would probably be best not to rely on them.

Reply:
Done.